typecheck error on `rest.v1.lists.$select().remove()` and others

I've observed a new typecheck error like the below since masto.js v7.7.0. It says the method requires 1-2 arguments but it should not be necessaryneeded.

You can reproduce the type error by running typecheck with test profile (I guess this config is not used on CI).

```ts
> tsc --project tsconfig.test.json --noEmit

...

tests/rest/v1/accounts.spec.ts:272:50 - error TS2554: Expected 1-2 arguments, but got 0.

272       await alice.rest.v1.lists.$select(list.id).remove();
                                                     ~~~~~~
...

tests/rest/v1/statuses.spec.ts:162:65 - error TS2554: Expected 1-2 arguments, but got 0.

162       let status = await bob.rest.v1.statuses.$select(statusId).reblog();
                                                                    ~~~~~~
```

It looks like https://github.com/neet/masto.js/pull/1359 introduced this type error first.

--- a/src/mastodon/rest/v1/lists.ts
+++ b/src/mastodon/rest/v1/lists.ts
@@ -76,7 +76,7 @@ export interface Lists$SelectResource {
    * @return N/A
    * @see https://docs.joinmastodon.org/methods/timelines/lists/
    */
-  remove: Method<void, string>;
+  remove: Method<void>;
 }
 
 export interface ListsResource {

--- a/src/mastodon/rest/v1/statuses.ts
+++ b/src/mastodon/rest/v1/statuses.ts
@@ -106,7 +106,7 @@ export type UpdateStatusParams = CreateStatusParams & {
 
 export interface ReblogStatusParams {
   /** any visibility except limited or direct (i.e. public, unlisted, private). Defaults to public. Currently unused in UI. */
-  readonly visibility: StatusVisibility;
+  readonly visibility?: StatusVisibility | null;
 }
 
 export interface TranslateStatusParams {
@@ -228,9 +228,9 @@ export interface Statuses$SelectResource {
   unmute: Method<Status>;
 
   /**
-   * Re-share a status.
+   * Boost a status
    * @return Status
-   * @see https://docs.joinmastodon.org/api/rest/statuses/#post-api-v1-statuses-id-reblog
+   * @see https://docs.joinmastodon.org/methods/statuses/#boost
    */
   reblog: Method<Status, ReblogStatusParams, HttpMetaParams<"json">>;
 

